lib/rubocop/config.rb

```diff
@@ -295,6 +296,15 @@ def target_ruby_version
         end
     end
 
+    def target_rails_version
+      @target_rails_version ||=
+        if for_all_cops['TargetRailsVersion']
```

Comment:
yeah I, with the intention of following the previously existing style, copied this directly from the `target_ruby_version` right above it. Then I deleted some of the `ruby-version` file checking and this can/should be simplified. I'd probably write it as 

```ruby
for_all_cops.fetch('TargetRailsVersion', DEFAULT_RAILS_VERSION)
```

but I'm happy to use `||` if that is preferred